src/words/ast/INodeAssign.java

```diff
@@ -8,9 +10,36 @@ public INodeAssign(Object... children) {
 		super(children);
 	}
 
+	/**
+	 * Always returns null. Throws WordsRuntimeException for called methods, including WordsObjectNotFoundException.
+	 * Side effect of assigning a property to an object.
+	 */
 	@Override
 	public ASTValue eval(WordsEnvironment environment) throws WordsRuntimeException {
-		// TODO
-		throw new AssertionError("Not yet implemented");
+		WordsObject obj = children.get(0).eval(environment).objValue;
```

Comment:
Yes that's right, we're using Assertions as a defensive coding practice for ourselves, not something the user should ever see.  if we trigger an AssertionError during our development, it means our assumptions about our code were wrong, or we've changed things such that those assumptions are no longer true, either of which will be good to have caught.